src/lib/jsonStatham.js

```diff
@@ -17,11 +17,10 @@ function ensureDataIsObject(data) {
 var jsonStatham = {
   setAdaptor(adaptor) { this.adaptor = adaptor },
   getAdaptor() { return (this.adaptor ? this.adaptor : new defaultAdaptor()) },
-  buildRequest(path, method, data, withCredentials) {
-    var headers = {} // TODO grab special header
+  buildRequest(path, method, data, withCredentials, additionalHeaders) {
```

Comment:
If additional headers are needed the logic should be outside of this library, so they should be passed fully formed.